Content.Server/Sound/BaseEmitSoundComponent.cs

```diff
@@ -11,8 +11,7 @@ namespace Content.Server.Sound
     /// </summary>
     public abstract class BaseEmitSoundComponent : Component
     {
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("sound")] public string? _soundName;
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("variation")] public float _pitchVariation;
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("soundCollection")] public string? _soundCollectionName;
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("variation")] public float PitchVariation { get; set; }
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("soundCollection")] public string? SoundCollectionName { get; set; }
```

Comment:
```suggestion
        [ViewVariables(VVAccess.ReadWrite)] [DataField("variation")] public float PitchVariation { get; set; } = 0.0f;
        [ViewVariables(VVAccess.ReadWrite)] [DataField("soundCollection", required: true)] public string SoundCollectionName { get; set; }
```

make soundCollection required, then remove all the 'is null' checks on it